lib/pbench/server/indexer.py

```diff
@@ -3588,6 +3588,7 @@ def __init__(self, idxctx, tbarg, tmpdir, extracted_root):
             )
         # Open the MD5 file of the tar ball and read the MD5 sum from it.
         md5sum = open("%s.md5" % (self.tbname)).read().split()[0]
+        self.md5sum = md5sum
```

Comment:
Perhaps add a comment that this is a convenience field since the `md5sum` value is stored already in `self.run_metadata["id"] at line 3472?

Reply:
Or perhaps I should just access the sub-fields directly. I went into the object with the intention of creating properties to store these; and somehow, even seeing that I was replicating existing data, at the time it didn't even occur to me to just back off and use them. Which, when I think about it now, was weird.